api/github.go

```diff
@@ -2,8 +2,8 @@ package api
 
 import (
 	"context"
-	
-	"vulnbot/logger"
+
+	"github.com/underdog-tech/vulnbot/logger"
```

Comment:
Yup! I was going to raise this at some point: why are we not using the full URIs for the module?